source/extensions/filters/http/ext_proc/processor_state.h

```diff
@@ -210,7 +210,7 @@ class ProcessorState : public Logger::Loggable<Logger::Id::ext_proc> {
   const envoy::config::core::v3::TrafficDirection traffic_direction_;
 
 private:
-  void clearRouteCache(const envoy::service::ext_proc::v3::CommonResponse& common_response);
+  virtual void clearRouteCache(const envoy::service::ext_proc::v3::CommonResponse&) PURE;
```

Comment:
nit: We probably can just have a default {} virtual function here instead of pure virtual.
Then we don' t need have it at all in `EncodingProcessorState`.

I don't feel strongly. It is up to you.

Reply:
sure, done!